src/Backend/OnboardingInsuranceAPI/Areas/Insurance/GetInsurance.cs

```diff
@@ -26,21 +26,23 @@ public async Task<InsuranceData> Handle(string pid)
         if (user is null)
             throw new ApiException(ErrorCode.NotFound);
 
-        return new InsuranceData
+        var insurance = await _context.Insurances.Where(i => i.Pid == pid).ToListAsync();
```

Comment:
If it is ToList.... the name of variable cannot be "insurance". It should be plural.

Reply:
Also AsNoTracking()